scipy/optimize/_qap.py

```diff
@@ -0,0 +1,785 @@
+import numpy as np
+import operator
+from . import (linear_sum_assignment, OptimizeResult)
+from .optimize import _check_unknown_options
+
+from scipy._lib._util import check_random_state
+import itertools
+
+
+def quadratic_assignment(A, B, method="faq", options=None):
+    r"""
+    Solve the quadratic assignment problem.
+
+    This function solves the Quadratic Assignment Problem (QAP) and the
+    Graph Matching Problem (GMP).
```

Comment:
```suggestion
    Approximates solution to the quadratic assignment problem and
    the graph matching problem.
```
This is the text that will appear in the `scipy.optimize` documentation, not the text in `__init__.py`.